pkgs/tools/security/safety/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, python3Packages }:
```

Comment:
generally the interpreter is what you want to import, in case you need to freeze some dependency version, this is done at the interpreter level.
```suggestion
{ stdenv, python3 }:
```

Reply:
the package set can be accessed with `python3.pkgs`